config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
You don't want to push `secrets.yml` to a public repo! Files with sensitive information (e.g., API keys) should be included in your `.gitignore`.